kvdb-rocksdb/src/lib.rs

```diff
@@ -443,15 +443,22 @@ impl Database {
 							stats_total_bytes += key.len();
 							batch.delete_cf(cf, &key).map_err(other_io_err)?
 						}
-						DBOp::DeletePrefix { col: _, prefix } => {
+						DBOp::DeletePrefix { col, prefix } => {
 							if prefix.len() > 0 {
-								let end_range = kvdb::end_prefix(&prefix[..]);
-								batch.delete_range_cf(cf, &prefix[..], &end_range[..]).map_err(other_io_err)?;
+								if let Some(end_range) = kvdb::end_prefix(&prefix[..]) {
+									batch.delete_range_cf(cf, &prefix[..], &end_range[..]).map_err(other_io_err)?;
+								} else {
+									for (key, _) in self.iter_from_prefix(col, &prefix[..]) {
+										batch.delete_cf(cf, &key[..]).map_err(other_io_err)?;
+									}
+								}
 							} else {
 								// Deletes all values in the column.
 								let end_range = &[u8::max_value()];
```

Comment:
Note that using [255, 255, 255... could make a smaller iteration (but would skip the test case).

Reply:
I think it's worth considering as a workaround or do you have other ideas?

Reply:
Not really, I will rework that tomorrow morning (will need to align the test case so it keeps testing the corner case).
What is the proper number of 255? 4 sounds ok

Reply:
The higher the number the better :) I was thinking more about 32 ;)

Reply:
I cut in the middle with 16.